pubspec.yaml

```diff
@@ -18,6 +18,7 @@ dependencies:
   stream_channel: '>=1.6.8 <3.0.0'
 
 dev_dependencies:
+  js: ^0.6.4
```

Comment:
This is used under `lib/`, so it should be in `dependencies` instead of `dev_dependencies`

Reply:
Ah, thank you!